### PR TITLE
Explore Page Anchor Links

### PR DESCRIPTION
**Pull request by gabbybruck**
I used gatsby-plugin-anchor-links for these links 
https://www.gatsbyjs.com/plugins/gatsby-plugin-anchor-links

Screen Shot 2022-05-20 at 8 37 51 PM
Screen Shot 2022-05-20 at 8 38 09 PM
Screen Shot 2022-05-20 at 8 38 38 PM
/

### CODE REVIEW

File: src/components/ExploreCarousel/index.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ import {
 import Carousel from "react-multi-carousel";
 import "react-multi-carousel/lib/styles.css";
 
-const ExploreCarousel = ({ section }) => {
+const ExploreCarousel = ({ section, id, key }) => {
```
Comment (mackenziekerwin) [inline src/components/ExploreCarousel/index.js]: can you remove the unused `key` param?

File: src/components/ExploreNav/styled.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+import Cursor from "../../images/cursor-pointer.svg";
+import { AnchorLink } from "gatsby-plugin-anchor-links";
+
+export const StyledLink = styled.div`
+  white-space: nowrap;
+  height: 3rem;
+  width: 100%;
+  transform: skew(15deg);
+  background-color: transparent;
+  cursor: url(${Cursor}) 20 20, pointer;
```
Comment (mackenziekerwin) [inline src/components/ExploreNav/styled.js]: nit: import `PointerStyles` from the styles file so you don't have to import the cursor image everywhere it's used; it replaces this whole line

File: src/components/ExploreNav/styled.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+export const CenterLink = styled(StyledLink)`
+  border-right: none;
+  border-left: none;
+`;
+
+export const RightLink = styled(StyledLink)`
+  border-left: none;
+`;
+
+export const LinkContainer = styled.div`
```
Comment (mackenziekerwin) [inline src/components/ExploreNav/styled.js:42]: can you make this sticky? I believe the designers intended the bar to be constant at the top as you scroll

File: src/components/ExploreNav/styled.js [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    margin-left: 33.33%;
+  }
+`;
+
+export const LinkText = styled(H3)`
+  margin: 0.6rem 0;
+  transform: skew(-15deg);
+  color: ${CREAM};
+  &:hover {
+    color: ${BLACK};
```
Comment (mackenziekerwin) [inline src/components/ExploreNav/styled.js:63]: can you move this to the `StyledLink` where you set the background color to pink? right now if you hover over the button but not the text itself it remains white

**Review by mackenziekerwin — COMMENTED** [review]
great!! just double check that it works with the view section once you merge main into this branch

File: src/components/ExploreNav/styled.js (hunk 4 above)
Comment (gabbybruck) [reply src/components/ExploreNav/styled.js:63]: for some reason when setting it in the StyledLink it isn't changing the color of the text, but I added a height and width to the LinkText and it seems to have solved the problem

Comment (mackenziekerwin) [reply src/components/ExploreNav/styled.js:63]: sounds good! I think adding `color: inherit` here would have also worked

File: src/components/ExploreNav/styled.js [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+  border-left: none;
+`;
+
+export const LinkContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  position: sticky;
+  top: 0;
+  z-index: 1;
+  margin-left: 5%;
```
Comment (mackenziekerwin) [inline src/components/ExploreNav/styled.js]: nit: if you change this to `margin: auto` you don't need to update it across breakpoints!!

**Review by mackenziekerwin — COMMENTED** [review]
agh part of me wants the full width of the background to be black so that it covers the background as you scroll and looks like a nav bar... I know that it will cover the headings when you click on the anchor link but I think that's ok. right now the transparency makes the button placement look unintentional
Screen Shot 2022-05-21 at 4 00 12 PM

so maybe wrap the whole container in a full-width, sticky div with a black background (and maybe like `1rem` of vertical padding)?

i know this is a super annoying request LOL it's lovely otherwise

**Review by mackenziekerwin — APPROVED** [review]
looks amazing!!

**Pull request merged**